### ISSUE TITLE
Show NewPipe as a target in all (more) "Open with..." scenarios

### ISSUE THREAD

**Issue by jmbreuer** [TeamNewPipe/NewPipe#5533]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

When clicking a YouTube link on Facebook, it originally opens in its own embedded browser. In its menu there is an "Open with..." option, which presents a choice of the system browser (Chrome in my case), the official YouTube app, and sharing targets published by Yatse to play or queue the video on a network-connected Kodi. 

NewPipe, however, does not show up there. 

I can use "Open with..." to go to the official YouTube app, and use "Share" from there - this results in a much larger list of possible targets including NewPipe. 

This is not limited to the Facebook app, but also others that work similarly. Telegram for sure (it shows exactly the same menu as Facebook, the function there is called "Open in...") and others like possibly reddit, Twitter, ... - the first two mentioned are those I can readily check right now. 

I would like for NewPipe to be included as a target from those apps / menus, just like Yatse puts Kodi targets there. 

#### Additional context

Screenshot showing the menu in question as it shows up for me:

![Screenshot_20210206-123435](https://user-images.githubusercontent.com/1792037/107117053-c88cce00-6877-11eb-9190-a7bbf84f7cbe.png)

The menu is scrolled up as far as it will go, showing all possible options. 

#### How will you/everyone benefit from this feature?

Easier use of NewPipe to play / background content that's shared and found in very typical places.

**Comment by TiA4f8R** [comment]
@jmbreuer It's more a bug than a feature request. Can you try to go to Android Settings, Apps, NewPipe, Open by default and see if Open links value is Never? If yes, can you try to set Open links value to Always Ask and try if NewPipe is now showing in your system resolver ("Open with")? Thank you in advance.

**Comment by jmbreuer** [comment]
"The plot thickens"...

I've looked at this setting, and NewPipe was already set to Always Ask.

Since I already was in this place, I also looked at the open links values of the google YouTube App, as well as Yatse. The google YouTube app had 'some defaults' set, and I also reset it to 'Always Ask'. Yatse already was set up for 'Always Ask'.

Already being right next to it, I also looked over the values of 'Supported Links' for all those Apps, and they all looked sensible to me.

Only after that, I tried to open some YouTube links using "Open with..." from previously problematic Apps, and lo and behold, this time NewPipe is shown as expected! :slightly_smiling_face: 

Since I did **not** change any of NewPipe's own defaults, this leaves me with two possible explanations at this time:

1. The fact that the YouTube App had some defaults set caused it to hide NewPipe, but now Yatse or Chrome
2. Something related to this changed leading up to NewPipe 0.20.11, which I have currently installed. Since I reported this on February 6th, it must have been the then-current version 0.20.8.

Anyway, I'm happy with how it is now; and I know one more place to look at regarding issues such as this. Closing this, I'll reopen / create a new issue if it should resurface beyond what's described here.
